README.md

```diff
@@ -1,70 +1,54 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Project Name
 
-## Available Scripts
+> This applications shows real-time data for cryptocurrencies current market pricing and activity. The data used is fetched from blockchain API.
 
```

Comment:
- [ ] Please use this new Microverse README [template ](https://github.com/microverseinc/readme-template), It will make the README file looks more professional.